packages/augur-test/src/tests/flash/ReasonableMarket.test.ts

```diff
@@ -0,0 +1,21 @@
+import { ACCOUNTS, FlashSession, addScripts, addGanacheScripts } from "@augurproject/tools";
```

Comment:
I renamed CannedMarkets to this because creating a single market is much faster than creating 20 plus many orders... and the original test was taking a very long time to run.